### DISCUSSION TITLE
Octopath Traveler

### DISCUSSION THREAD

**Discussion by Buricat** [skyline-emu/skyline#504] [Game Requests]
This game can be a good start to check if HD-2D will work fine on skyline. This is one of the games I really wanted to try if I have a real Switch device.

**Comment by PixelyIon** [comment]
This is a Unreal Engine game and that whole engine is full of doing fairly complex stuff, I do want Octopath Traveler to work eventually but it's going to take a while.

**Comment by PixelyIon** [comment]
> This is one of the games I really wanted to try if I have a real Switch device.
I should additionally add that you **do** need a Switch to use Skyline as there's no other way to dump games legally. I realized this is a duplicate of #475 and I'll be locking then deleting this as a result.
